src/resources/baseResource.ts

```diff
@@ -113,14 +136,73 @@ export class BaseResource {
         } as ListYieldReturn<T>)
     );
 
-    const response = Object.assign(first, {
+    return Object.assign(first, {
       [Symbol.asyncIterator]: this.listIterator.bind(
         this,
         listParams
       ) as () => AsyncGenerator<T, undefined>,
     });
+  }
 
-    return response;
+  protected _find<T>({
+    path,
+    responseSchema,
+    overrides,
+  }: FindParams<T>): Promise<ItemResponse<T>> {
+    return this.apiClient.request<ItemResponse<T>>(
+      {
+        method: 'GET',
+        path,
+        overrides,
+      },
+      {
+        responseSchema,
+      }
+    );
+  }
+
+  private payloadRequest<T>(
```

Comment:
i think this has a very small chance of ending up being too catch all but we'll see